integration/cri-o/cri-o.sh

```diff
@@ -32,18 +33,24 @@ else
 	popd
 fi
 
-OLD_IFS=$IFS
-IFS=''
-
 # Skip CRI-O tests that currently are not working
 pushd $GOPATH/src/${crio_repository}/test/
 for i in ${skipCRIOTests[@]}
 do
 	sed -i '/'${i}'/a skip \"This is not working (Issue https://github.com/kata-containers/agent/issues/138)\"' "$GOPATH/src/${crio_repository}/test/ctr.bats"
 done
 
-IFS=$OLD_IFS
-
+# By default run CRI-O tests using devicemapper
+MAJOR=$(echo "$VERSION_ID"|cut -d\. -f1)
 export STORAGE_OPTIONS="--storage-driver devicemapper --storage-opt dm.use_deferred_removal=false"
+
+# But if on ubuntu 17.10 or newer, test using overlay
+# This will allow us to run test with at least 2 different
+# storage drivers.
+if [ "$ID" == "ubuntu" ] && [ "$MAJOR" -ge 17 ]; then
+	export CRIO_STORAGE_DRIVER_OPTS="--storage-driver=overlay"
```

Comment:
nit: replace `--storage-driver=overlay` with `--storage-driver overlay` since that's the syntax used for devicemapper.